packages/bot-web-ui/src/stores/journal-store.js

```diff
@@ -53,9 +46,21 @@ export default class JournalStore {
         { id: message_types.NOTIFY, label: localize('Notifications') },
         { id: message_types.SUCCESS, label: localize('System') },
     ];
+    journal_filters = [];
+    unfiltered_messages = [];
+
+    restoreStoredJournals() {
+        this.journal_filters = getSetting('journal_filter') || this.filters.map(filter => filter.id);
+        this.unfiltered_messages = getStoredItemsByUser(this.JOURNAL_CACHE, this.core?.client.loginid, []);
```

Comment:
minor improvement: we could use destructurization to improve readability 😌
```suggestion
const { loginid } = this.core?.client;
this.unfiltered_messages = getStoredItemsByUser(this.JOURNAL_CACHE, loginid, []);

```